moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -216,6 +218,13 @@ const ProjectTeamTable = ({
     <ApolloErrorHandler errors={error}>
       <MaterialTable
         columns={columns}
+        components={{
+          EditRow: (props, rowData) => <MTableEditRow {...props} onKeyDown={(e) => {
+              if (e.keyCode === 13) {
+                // Bypass default MaterialTable behavior of submitting the entire form when a user hits enter
```

Comment:
I'm kinda confused how this is working since no code executes in the block for keycode 13. But it must be doing something because the deploy instance looks good. If there is any links to Github Issues or Docs that would help someone understand this `EditRow` API and what default behavior it overrides, I think that could help.

Reply:
From what I understand, this just clears the default behavior so that nothing occurs when a user presses enter other than that a value is selected. I thought about putting a `return` statement or `e.preventDefault()` but as it was working as-is I didn't want to potentially block any desired behavior. Happy to try that out if we don't want to have an empty conditional. In case it is helpful, I pulled the workaround from this [MaterialTable PR comment](https://github.com/mbrn/material-table/pull/2008#issuecomment-662529834). The first part of the comment is for enter behavior and includes the empty conditional, the second part would be for a user hitting escape and canceling the action.